peerconnection.go

```diff
@@ -854,6 +854,10 @@ func (pc *PeerConnection) startRTPReceivers(incomingTracks map[uint32]trackDetai
 		for i := range localTransceivers {
 			t := localTransceivers[i]
 
+			if t.Mid() != incoming.mid {
+				continue
```

Comment:
I believe this will skip available transceivers that have never been used for sending. That's why the `TestPeerConnection_Renegotation_AddTrack_Multiple` test is failing

Reply:
I think we'll need to do the same thing we did in `generateMatchedSDP` when `includeUnmatched` is set to `true`.

Reply:
@jeremija Yeah, the real issue is that, when receiving an offer we're starting receivers in SetRemoteDescription and assigning transceiver mid in CreateAnswer (so later). We should follow JSEP (https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-26#section-5.10) and create assign transceivers to a media section in SetRemoteDescription

I reported this in #1171 and proposed PR #1179. Perhaps this should go after #1179.

Reply:
@sgotti Yeah I was just thinking about that other PR of yours. I'm having a look at it now.

Reply:
@jeremija Now that #1179 has been merged this should be ok.